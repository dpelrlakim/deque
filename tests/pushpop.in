newInit a
push_back a 23
push_back a 29
push_back a 31
[] a 9
pop_front a
pop_front a
pop_back a
[] a 7
newDefault dq2
= a dq2
push_front a 1
push_front a 0
push_front a -1
pop_back a
pop_back a
pop_back a
