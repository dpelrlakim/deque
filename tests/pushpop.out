a: 2 3 5 7 11 13 17 19
a: 2 3 5 7 11 13 17 19 23
a: 2 3 5 7 11 13 17 19 23 29
a: 2 3 5 7 11 13 17 19 23 29 31
29
a: 2 3 5 7 11 13 17 19 23 29 31
a: 3 5 7 11 13 17 19 23 29 31
a: 5 7 11 13 17 19 23 29 31
a: 5 7 11 13 17 19 23 29
29
a: 5 7 11 13 17 19 23 29
a: 5 7 11 13 17 19 23 29
dq2:
a:
dq2:
a: 1
dq2:
a: 0 1
dq2:
a: -1 0 1
dq2:
a: -1 0
dq2:
a: -1
dq2:
a:
dq2:
